near-sdk-macros/src/core_impl/info_extractor/impl_item_method_info.rs

```diff
@@ -28,3 +28,46 @@ impl ImplItemMethodInfo {
         }
     }
 }
+
+// Rustfmt removes comas.
+#[rustfmt::skip]
+#[cfg(test)]
+mod tests {
+    use syn::{parse_quote, Type, ImplItemMethod};
+    use crate::core_impl::ImplItemMethodInfo;
+
+    #[test]
+    fn init_no_return() {
```

Comment:
are we supporting an `init` function that returns a `Result`? if so, maybe another test to test here would also be worthwhile